### PR TITLE
Meraki scenario guide - Changed section markup types

### PR DESCRIPTION
**Pull request by kbreit**
##### SUMMARY
Changed section break markup types

@acozine - This is related to your request in PR 45494 (https://github.com/ansible/ansible/pull/45494#discussion_r220275329)

##### ISSUE TYPE
- Docs Pull Request

##### COMPONENT NAME
meraki

##### ANSIBLE VERSION
```
ansible 2.8.0.dev0 (meraki/scenario-fixes 9f5f26bc46) last updated 2018/09/26 07:34:57 (GMT -500)
  config file = None
  configured module search path = ['/Users/kbreit/.ansible/plugins/modules', '/usr/share/ansible/plugins/modules']
  ansible python module location = /Users/kbreit/Documents/Programming/ansible/lib/ansible
  executable location = /Users/kbreit/Documents/Programming/ansible/bin/ansible
  python version = 3.5.4 (default, Feb 25 2018, 14:56:02) [GCC 4.2.1 Compatible Apple LLVM 9.0.0 (clang-900.0.39.2)]

```

### CODE REVIEW

File: docs/docsite/rst/scenario_guides/guide_meraki.rst [hunk 1]
```diff
@@ -1,33 +1,34 @@
...
 
 
 .. _meraki_guide_intro:
 
 What is Cisco Meraki?
----------------------
+=====================
 
-Cisco Meraki is an easy to use, cloud based, network infrastructure platform for enterprise environments. While most network hardware uses command line interfaces (CLIs) for configuration, Meraki uses an easy to use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
+Cisco Meraki is an easy to use, cloud-based, network infrastructure platform for enterprise environments. While most network hardware uses command-line interfaces (CLIs) for configuration, Meraki uses an easy-to-use Dashboard hosted in the Meraki cloud. No on-premises management hardware or software is required - only the network infrastructure to run your business.
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_meraki.rst]: there's one more "easy-to-use" here, right at the beginning of this line

File: docs/docsite/rst/scenario_guides/guide_meraki.rst [hunk 2]
```diff
@@ -1,33 +1,34 @@
 .. _meraki_guide:
 
+******************
 Cisco Meraki Guide
-==================
+******************
```
Comment (acozine) [inline docs/docsite/rst/scenario_guides/guide_meraki.rst:6]: If you think the guide will grow, a local TOC might be helpful here. The syntax is:
```
.. contents::
   :local:
```
It's not required, but try it out while you're making the other change and see what you think.

**Review by acozine — COMMENTED** [review]
Thanks for addressing my comments from #45494. I'll merge that one and look for a final version on this and a backport.

**Review by acozine — CHANGES_REQUESTED** [review]
@kbreit that second easy-to-use is a blocker to merging this . . . the local TOC is optional - try it and see what you think.

File: docs/docsite/rst/scenario_guides/guide_meraki.rst (hunk 1 above)
Comment (kbreit) [reply docs/docsite/rst/scenario_guides/guide_meraki.rst]: Fix made on my iPad so I can’t validate but once it’s tested, it should be good.

**Review by acozine — APPROVED** [review]

**Comment by acozine** [comment]
Thanks @kbreit - I'll backport this to 2.7.

**Pull request merged**